src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -95,9 +95,12 @@ private static void ReplaceProjectJsonTemplateValues(string projectDirectory)
         {
             string projectJsonFile = Path.Combine(projectDirectory, "project.json");
 
-            File.Move(
-                Path.Combine(projectDirectory, "project.json.template"),
-                projectJsonFile);
+            if(File.Exists(Path.Combine(projectDirectory, "project.json.template")))
```

Comment:
Maybe just a single call to `Path.Combine(projectDirectory, "project.json.template")` and put it in `string projectJsonTemplateFile` variable?